tests/h/search/query_test.py

```diff
@@ -707,21 +707,24 @@ def test_matches_tags(self, search, Annotation):
 
         assert sorted(result.annotation_ids) == sorted(matched_ids)
 
-    def test_ors_any_matches(self, search, Annotation):
+    def test_ands_any_matches(self, search, Annotation):
         """
-        Any is expected to match any of the following fields;
+        Any is expected to match all of the following fields;
         quote, text, uri.parts, and tags
         that contain any of the passed keywords.
         """
         Annotation(target_selectors=[{'exact': 'selected baz text'}])
```

Comment:
On second thought..I think we could make this test a lot simpler/faster by removing a lot of these extra Annotations that aren't expected to be found. Can you remove the one's that aren't needed/repeats?

Reply:
Yeah, why not.